tensorflow/lite/micro/kernels/neg_test.cc

```diff
@@ -71,6 +71,72 @@ void TestNegFloat(std::initializer_list<int> input_dims_data,
   }
 }
 
+// input_quantization_buffer: buffer used for the quantization data
+void TestNegQuantizedInt8(float* input, int8_t* quantized_input_data,
+                          float input_min, float input_max,
+                          float* expected_output,
+                          int8_t* quantized_expected_output_data,
+                          int8_t* quantized_output_data, float output_min,
+                          float output_max,
+                          std::initializer_list<int> dimension_data) {
+  TfLiteIntArray* tensor_dims = IntArrayFromInitializer(dimension_data);
+  const int element_count = ElementCount(*tensor_dims);
+  constexpr int inputs_size = 1;
+  constexpr int outputs_size = 1;
+  constexpr int tensors_size = inputs_size + outputs_size;
+
+  // quantize input
+  std::transform(input, input + element_count, quantized_input_data,
```

Comment:
can you add the test in tensorflow/lite/kernels as well (also the kernel).

FYI, micro and tflite shares the most kernel implementation.